backend/backend/api/routes.py

```diff
@@ -130,12 +130,12 @@ async def read_root():
     </style>
```

Comment:
<begin comment>Consider renaming the buttons from \u2018Test\u2019 and ‘Tell Joke’ to more descriptive names, such as ‘Research’ and ‘Generate Joke’, respectively. This will improve user understanding of their functionality.<end comment>